controllers/etcdcluster_controller_test.go

```diff
@@ -267,8 +266,23 @@ func (s *controllerSuite) testClusterController(t *testing.T) {
 			err = try.Eventually(func() error {
 				var actual v1.PersistentVolumeClaimList
 				err := s.k8sClient.List(s.ctx, &actual, client.InNamespace(etcdCluster.Namespace))
+
+				// Since we can't control when PVCs with the pvc-protection are deleted
+				// we will remove them from this list
+				var pvcsWOProtectionFinalizer v1.PersistentVolumeClaimList
```

Comment:
It's probably worth having an extra close look at this, potentially we can sove this some other way

Reply:
Could you please describe the main issue?